android/quest/src/main/java/org/smartregister/fhircore/quest/ui/usersetting/UserSettingInsightScreen.kt

```diff
@@ -98,3 +219,166 @@ fun UserSettingInsightScreen(
     }
   }
 }
+
+@Composable
+fun UnsyncedDataView(
+  first: String,
+  second: String,
+) {
+  Box(
+    Modifier.fillMaxWidth().padding(16.dp),
+  ) {
+    Text(
+      text = first,
+      modifier = Modifier.align(Alignment.CenterStart).testTag(INSIGHT_UNSYNCED_DATA),
+      fontSize = 16.sp,
+      color = colorResource(id = R.color.grayText),
+      fontWeight = FontWeight.Medium,
+    )
+    Text(
+      modifier = Modifier.align(Alignment.CenterEnd),
+      text = second,
+      color = LoginDarkColor,
+      fontWeight = FontWeight.Bold,
+    )
+  }
+}
+
+@Composable
+fun InsightInfoView(
+  title: String,
+  headingOne: String?,
+  contentOne: String?,
+  headingTwo: String?,
+  contentTwo: String?,
+  headingThree: String?,
+  contentThree: String?,
+  headingFour: String?,
+  contentFour: String?,
```

Comment:
This can be refactored to accept any `n` number of items. The header + content information can be captured in a `Pair` class as key (header) and value(content). Have the items in a of Pairs LIST<Pair>. Iterate through the list to render the desired content. (The field properties like color can be configured). If need be you may create a new data class to capture the required items if a `Pair` class will not suffice.